web/src/integration_url_modal.ts

```diff
@@ -57,6 +75,38 @@ export function show_generate_integration_url_modal(api_key: string): void {
             );
         });
 
+        function render_config(config: ConfigOption[]): void {
+            const validatedConfig = ConfigOptionsSchema.parse(config);
+            const $configContainer = $("#integration-url-config-options-container");
```

Comment:
Please use underscore for separators in variable names, as that's our code style. (camelCase is reserved for class names in our frontend).